src/transformers/models/roberta_prelayernorm/configuration_roberta_prelayernorm.py

```diff
@@ -0,0 +1,81 @@
+# coding=utf-8
+# Copyright 2022 The Google AI Language Team Authors and The HuggingFace Inc. team.
+# Copyright (c) 2018, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" RoBERTa-PreLayerNorm configuration"""
+from collections import OrderedDict
+from typing import Mapping
+
+from ...onnx import OnnxConfig
+from ...utils import logging
+from ..bert.configuration_bert import BertConfig
```

Comment:
Pretty sure we had a HackToBertFest sprint to remove dependencies with `Bert` wherever we can, see #19303. So we should be implementing from scratch and use `#copied_from` wherever possible

Reply:
Not sure why this was copied that way. Regardless, it is updated now.